Options/Applicative/Help/Core.hs

```diff
@@ -59,12 +59,13 @@ optDesc pprefs style info opt =
         = mempty
         | isEmpty chunk || not (descSurround style)
         = mappend chunk suffix
-        | hinfoDefault info
-        = mappend (fmap brackets chunk) suffix
+        --  | hinfoDefault info
+        -- = mappend chunk suffix
         | null (drop 1 descs)
         = mappend chunk suffix
         | otherwise
-        = mappend (fmap parens chunk) suffix
+        = mappend chunk suffix
```

Comment:
These 3 cases have the same result logic.
Please double check, and remove the guarded expressions if you're sure.

Reply:
Incorporated changes you've suggested below